### PR TITLE
feat: add policy parameter to evaluate command

### PR DESCRIPTION
**Pull request by ls-ross-macduff**
Also changed the command to "image evaluate" to make it consistent with the documentation.

### CODE REVIEW

**Comment by ls-ross-macduff** [comment]
I made this PR against the Alpha branch per the contribute instructions but since that branch didn't have the config I wanted to update I had to copy the whole thing in. Please let me know if you'd prefer this against master (or another branch).

**Comment by afiune** [comment]
@ls-ross-macduff Thank you so much for your contribution, you did the right thing opening your PR against Alpha but for some reason that branch is not on sync with `master`, we are going to fix that and then review this change. We will get back to you shortly.

**Comment by afiune** [comment]
Fixing Alpha so we can run our automated tests against your PR. Hold tight! 🤞🏽

Screen Shot 2022-03-22 at 12 38 57 PM

File: src/commands/inline-scanner-evaluate.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ parameters:
     type: boolean
     default: false
   save:
-    description: Save results in Lacework platform
+    description: Save results in the Lacework platform
```
Comment (afiune) [inline src/commands/inline-scanner-evaluate.yml]:
```suggestion
    description: Save results in Lacework platform
```

**Review by afiune — APPROVED** [review]
tenor-12657430

**Comment by ls-ross-macduff** [comment]
Thank you for merging this to Alpha. Looking forward to this being available in an upcoming release 😄

**Pull request merged**